spatial-lib/src/cmr/spatial/encoding/gmd.clj

```diff
@@ -74,3 +93,44 @@
   ;; single gml geometry element
   (let [polygons (-> bounding-polygon-element (cx/elements-at-path [:polygon]))]
     (map #(-> % :content first gml/decode) polygons)))
+
+
+(defmethod decode-geo-content :EX_GeographicDescription
+  [geo-desc]
+  (let [orbit-str (cx/string-at-path geo-desc [:geographicIdentifier :MD_Identifier :code :CharacterString])
+        description-type  (cx/string-at-path geo-desc [:geographicIdentifier :MD_Identifier :description :CharacterString])
+        ascending-crossing (util/get-index-or-nil orbit-str "AscendingCrossing:")
+        start-lat (util/get-index-or-nil orbit-str "StartLatitude:")
+        start-direction (util/get-index-or-nil orbit-str "StartDirection:")
+        end-lat (util/get-index-or-nil orbit-str "EndLatitude:")
+        end-direction (util/get-index-or-nil orbit-str "EndDirection:")]
+    (when (= description-type "OrbitParameters")
+      (orbit/->Orbit
+       (when ascending-crossing
+         (let [asc-c (subs orbit-str
+                           ascending-crossing
+                           (or start-lat start-direction end-lat end-direction
+                               (count orbit-str)))]
+           (Float. (str/trim (subs asc-c (inc (.indexOf asc-c ":")))))))
```

Comment:
I believe these calls to `Float.` will throw an exception if the value can't be represented as a float. I see validation functions on the UMM, but this parsing happens before converting to the model.

Can you add a test with invalid ISO-SMAP metadata to verify we successfully return an error to the client and not an internal server error?